src/queries/text.js

```diff
@@ -9,6 +9,31 @@ import {
   buildQueries,
 } from './all-utils'
 
+function ignoreHiddenElements(container, selector, baseArray) {
+  const hiddenElements = [
+    ...Array.from(container.querySelectorAll("[aria-hidden='true']")),
+  ]
+  baseArray.forEach(base => {
+    if (base.getAttribute('aria-hidden') === 'true') {
+      hiddenElements.push(base)
+    }
+  })
+  let foundHiddenChildren = [...hiddenElements]
+  hiddenElements.forEach(hiddenElement => {
+    foundHiddenChildren = [
+      ...foundHiddenChildren,
+      ...Array.from(hiddenElement.querySelectorAll(selector)),
+    ]
+  })
```

Comment:
Rather than need a `let` and doing reassignment, we can do this all in one go with the power of `flatMap`:
```suggestion
  const foundHiddenChildren = [
    ...hiddenElements,
    ...hiddenElements.flatMap(hiddenElement => Array.from(hiddenElement.querySelectorAll(selector))),
  ]
```

Reply:
let is fine